internal/kafka/internal/services/data_plane_kafka.go

```diff
@@ -452,6 +455,9 @@ func (d *dataPlaneKafkaService) getManagedKafkaStatus(status *dbapi.DataPlaneKaf
 				return statusReady
 			}
 			if strings.EqualFold(c.Status, "Unknown") {
+				if strings.EqualFold(c.Reason, "Installing") {
+					return statusResuming
```

Comment:
Added the new `resuming` state here. In this case though, we don't do anything since the Kafka should already have status `resuming`. We'll only log it as seen in L#207 above.

Reply:
Looks like this is no longer the case for `Resuming`. FSO will return Ready condition with `Status=False, Reason=Installing`. Will remove this.